### ISSUE TITLE
Folder/File does not exist

### ISSUE THREAD

**Issue by RagTMS** [gekware/minecraft-server-hibernation#135]
I feel like I'm doing something silly but I can't get the server to work

This is what the exe says

``` go
╔═════════════════════════════════════════════╗
║  _ __ ___  ___| |__                         ║
║ | '_ ` _ \/ __| '_ \                        ║
║ | | | | | \__ \ | | |                       ║
║ |_| |_| |_|___/_| |_| v2.4.4                ║
║ Copyright (C) 2019-2021 gekigek99           ║
║ github: https://github.com/gekigek99        ║
║ remember to give a star to this repository! ║
╚═════════════════════════════════════════════╝
2022/03/10 20:42:20 [←[0;34minfo←[0m  *   ] checking OS support...
2022/03/10 20:42:20 [←[0;34minfo←[0m  *   ] loading config default...
2022/03/10 20:42:20 [←[0;34minfo←[0m  *   ] loading config runtime...
2022/03/10 20:42:20 [←[0;31merror ←[0m*** ] main: LoadConfig: checkConfigRuntime: specified server file/folder does not exist:

```

and here is my config
``` yaml
{
  "Server": {
    "Folder": "D:/Raghu/Minecraft Extra/Servers/SMP",
    "FileName": "server.jar",
    "Protocol": 756,
    "Version": "1.18.2"
  },
  "Commands": {
    "StartServer": "java -Xmx8G -jar server.jar nogui
PAUSE",
    "StartServerParam": "-Xmx8192M -Xms2048M",
    "StopServer": "stop",
    "StopServerAllowKill": 60
  },
  "Msh": {
    "Debug": 1,
    "InfoHibernation": "                   §fserver status:\n                   §b§lHIBERNATING",
    "InfoStarting": "                   §fserver status:\n                    §6§lWAKING UP",
    "NotifyUpdate": true,
    "ListenPort": 25555,
    "TimeBeforeStoppingEmptyServer": 30
  }
}

```

I put {} around the Folder and FileName and switched the / to \ but it says the same thing. The server is fabric 1.18.2.

Thank you

**Comment by RagTMS** [comment]
Hi, I changed the config to "Folder": "D://Raghu//Minecraft Extra//Servers//SMP" but it still says the same message 
and when I try to do the 
cd "D:/Raghu/Minecraft Extra/Servers/SMP"
./msh.exe
it says this

D:\Raghu\Minecraft Extra\Servers\SMP\minecraft-server-hibernation>./msh.exe
'.' is not recognized as an internal or external command,
operable program or batch file.

Can you send a link to your discord channel so I can join it please,

Thank you

**Comment by gekigek99** [comment]
[discord server](https://discord.gg/mBSnEypcYC)

> it says this
> ```
> D:\Raghu\Minecraft Extra\Servers\SMP\minecraft-server-hibernation>./msh.exe
> '.' is not recognized as an internal or external command, operable program or batch file.
> ```

ok maybe try 
```
cd "D:/Raghu/Minecraft Extra/Servers/SMP"
msh.exe
```

if it doesnt work:
i think i fixed this in 8d85e478e7949599f507330c0db47a9d724f6e27
here is the [compiled binary](https://github.com/gekware/minecraft-server-hibernation/files/8236497/msh.zip) of this commit
